### DISCUSSION TITLE
Collaboration Invitation: Join Global Health AI

### DISCUSSION THREAD

**Discussion by Aditya-9716** [medplum/medplum#5184] [General]
Hi Medplum team,
We’ve been following your excellent work , and we see exciting opportunities for collaboration with our open-source community, [Global Health AI](https://github.com/GlobalHealthAI). Our mission is to leverage AI to tackle global healthcare challenges, and we’d love to work together on projects that can drive real-world impact.
We are actively seeking contributors for:
•	AI models to improve healthcare diagnostics and access
•	Ethical frameworks for responsible AI in healthcare
•	Working with healthcare data to develop scalable solutions
Would love to discuss how we could collaborate! You can check out our repositories here: [Global Health AI](https://github.com/GlobalHealthAI)
Looking forward to exploring this further!
Best regards,
Aditya Pal
[GitHub Handle](https://github.com/Aditya-9716rl)

**Comment by codyebberson** [comment]
Congratulations on your initiative, and best of luck!
